### PR TITLE
Update the mega linter because of GitHub Action deprecations

### PR DESCRIPTION
**Pull request by hawkowl**
### Which issue this PR addresses:
Chore from warnings in github

### What this PR does / why we need it:

Fixes up deprecation warnings

### Test plan for issue:

It is tests :)

### Is there any documentation that needs to be updated for this PR?

N/A

### CODE REVIEW

File: .mega-linter.yml [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
 APPLY_FIXES: none
-# ENABLE:  # If you use ENABLE variable, all other languages/formats/tooling formats will be disabled by default
 ENABLE_LINTERS:
-  - YAML_YAMLLINT # If you use ENABLE_LINTERS variable, all other linters will be disabled by default
+  - YAML_YAMLLINT
 EXCLUDED_DIRECTORIES:
-  - vendor
+  - "vendor"
+  - "node_modules"
 YAML_YAMLLINT_CONFIG_FILE: .yaml-lint.yml
```
Comment (SudoBrendan) [inline .mega-linter.yml:10]: just noticing that the file edited in this PR and this reference do not align. Am I missing something or is this a typo from a previous commit? The new linter seems to use a different default, one that would work if we actually removed this "custom" one: https://megalinter.io/latest/descriptors/yaml_yamllint/

**Review by SudoBrendan — COMMENTED** [review]
Changes look good, I wonder if we need to modify the configured ref to the yamllint config file path while using the new linter?

File: .mega-linter.yml (hunk 1 above)
Comment (hawkowl) [reply .mega-linter.yml:10]: the path is still different, dunno why we have that name in particular. I'm not too keen to change it if it works :)

Comment (hawkowl) [reply .mega-linter.yml:10]: OH, right. This yaml-lint is for the yaml-lint config in the proj root dir, the one changed in the PR is for the yaml linting GH action.

**Review by ellis-johnson — APPROVED** [review]
LGTM!

**Pull request merged**